wsi/drm/drmdisplaymanager.cpp

```diff
@@ -346,11 +346,13 @@ bool DrmDisplayManager::UpdateDisplayState() {
     NotifyClientsOfDisplayChangeStatus();
   }
 
+#ifdef NESTED_DISPLAY_SUPPORT
```

Comment:
This is not the right fix. We need to add this in iahwc2 side and not register hot plug callback in their.

Reply:
OK But the NESTED_DISPLAY flag is not added in all the places its purposefully done like that.